test/test.js

```diff
@@ -38,5 +39,19 @@ require('../glpk.js').then(glpkjs => {
 		})
 		t.end();
 	});
+  
+  tape('The time limit should kill the solver before finding optimal solution', { timeout: 99999 }, t => {
+    const problem = 'mip2'
+    let json = JSON.parse(fs.readFileSync(`${__dirname}/data/${problem}.json`).toString());
+    const settings = _.defaults({tmLim: 1}, json.settings)
```

Comment:
should we find a way to test the mipGap also?

Reply:
For examining `mipGap` we should have a complicated (big) problem. I don't know if it's a good idea to have it here in the public repo (given that we need to generate a big hard problem).
probably it's better to test it while implementing it in the matcher along with other parameters.